kernel/comps/virtio/src/device/block/mod.rs

```diff
@@ -111,6 +111,12 @@ pub struct VirtioBlockTopology {
     opt_io_size: u32,
 }
 
+#[derive(Debug, Copy, Clone)]
+#[repr(C)]
+pub struct VirtioBlockFeature {
+    flush: bool,
```

Comment:
Rename it `can_flush` or `support_flush` to make it more obvious that the field is a boolean value.